driver-core/src/main/java/com/datastax/driver/core/ResultSet.java

```diff
@@ -39,7 +34,11 @@
  * <p/>
  * Note that this class is not thread-safe.
  */
-public interface ResultSet extends Iterable<Row> {
+public interface ResultSet extends PagingIterable<ResultSet, Row> {
+
+    // redeclared only to make clirr happy
+    @Override
+    Row one();
```

Comment:
Since this is going into 3.1.0, does it matter that clirr isn't compliant for this particular case?